Issue-tracking documentation language reworked (#398)
ubports/docs.ubports.com: about/process/issue-tracking.rst

repository. This means, that issues of almost all components that are
distributed as with the system-image are tracked in the
`Ubuntu Touch tracker <https://github.com/ubports/ubuntu-touch>`__. An
exception of this are click-apps that can be updated independently through
Since quality assurance depends heavily community, issues are
tracked where users expects them, instead of separated by repository.
This means issues of almost all distributed components (as with the system-image)
are tracked in the `Ubuntu Touch tracker <https://github.com/ubports/ubuntu-touch>`__.
An exception to this are click-apps, which can be updated independently through
the OpenStore.

Most other repositories track issues locally. If you're unsure whether a
repository uses its own tracker or not, consult the README.md file.
Repositories that don't track issues locally have their bugtracker disabled.
Most other repositories track issues locally. You will find out whether a
repository uses its own tracker or not in its README.md file.
Repositories that don't track issues locally have their bugtracker turned off.

This page is mainly about the Ubuntu Touch tracker, but most principles apply

> to other projects as well.

Author: I was thinking "too", but then "as well" with _two_ words. The horrors.

>      Practical exceptions to purity might apply and must (no exceptions) be described in the project's README.md file.

Author: If you do fluff, you can do intrigue.

>      Practical exceptions to purity might apply and must (no exceptions) be described in the project's README.md file.

Reviewer: i dont understand

>      Practical exceptions to purity might apply and must (no exceptions) be described in the project's README.md file.

Author: It is a bit of a joke, somewhat a conundrum. There is no exclamation mark thrown at the reader. It is fine.

>      Practical exceptions to purity might apply and must (no exceptions) be described in the project's README.md file.

Reviewer: We're talking past each other. My original comment was that in my opinion the original wording was clearer. The use of the word purity was clear as it was introduced to contrast with practicality. In the new proposal, it seems that the word purity is just falling from the sky. The original phrasing was light hearted and intuitively clear. The new phrasing is light hearted but confusing. Also should it be 'may' not 'might'?. Lastly and most importantly, I disagree with the new content! That exceptions **must** be described. I think this is the opposite of the original statement about Practicality beats purity. And not only are we turning around the meaning, we are extra, extra emphasizing it with an additional "no exceptions" -- geez. chill. why the urgency? Even if I would agree that it's a good ideal to be zealously pure here - it's just unrealistic. In the real world it will never happen that all exceptions are diligently documented. And I don't think they should have to be! Because, you know what, Practicality **does** beat purity. So, I maintain: Please revert.

If problems are found, the issue is moved back to "Accepted".
* **Release Candidate**: A patch passing QA, ready for release. In case of DEB packages included in the system-image, the patch will be included in the next over-the-air update on the `rc` channel, and (provided  everything goes well) in the next release of the `stable` channel.
* **None (removed from the project)**: Open issue labeled "help wanted". Community contributions are required to resolve it. If it's closed, either a patch has been released on the stable channel (a comment on the issue should link to the patch) or the issue is rejected (labeled "wontfix").

Labels
------

All issues — even closed ones — should be labeled to allow use of GitHub's
global filtering. For example, `these are all of the issues labeled 'enhancement' inside @ubports <https://github.com/search?utf8=%E2%9C%93&q=is%3Aopen+org%3Aubports+label%3A%22feature+request%22&type=>`_. Consult the `GitHub help pages <https://help.github.com/articles/about-searching-on-github/>`__ for more info on searching and filtering.

List of labels normally used by all repositories:

- **needs confirmation**: The bug needs confirmation and / or further

>   info from affected users.

Author: Good call

Issue-Tracking Guidelines
=========================

This document describes the standard process of dealing with new issues in UBports projects.
(Not to be confused with the :doc:`guide on writing a good bugreport </contribute/bugreporting>`.)

Where are bugs tracked?
-----------------------

Since quality assurance depends heavily on community effort, issues are

> tracked where users expects them, instead of separated by repository.

Reviewer: expect

> tracked where users expects them, instead of separated by repository.

Author: ```suggestion
tracked where users expect them, instead of separated by repository.
```

This means issues of almost all distributed components (as with the system-image)
are tracked in the `Ubuntu Touch tracker <https://github.com/ubports/ubuntu-touch>`__.
An exception to this are click-apps, which can be updated independently through
the OpenStore.

Most other repositories track issues locally. You will find out whether a
repository uses its own tracker or not in its README.md file.
Repositories that don't track issues locally have their bugtracker turned off.

This page is mainly about the Ubuntu Touch tracker, but most principles apply
to other projects as well.

.. note::

>      Practical exceptions to purity might apply and must (no exceptions) be described in the project's README.md file.

Author: ```suggestion
     Practical exceptions to purity are to be described in the project's README.md file.
```

GitHub projects
---------------

In the interest of transparency and communication, GitHub projects (Kanban-Boards)
are used wherever practical. In case of github.com/ubports/ubuntu-touch, a
single project is used for all issues. Projects support filtering by labels,
so that only issues belonging to a specific team or ones affecting a specific
device can be viewed.

These are the standard columns:

* **None (awaiting triage)**: Issue approved by a member of the QA team awaiting review from the responsible development team.

> * If a bug, instructions to reproduce are included in the issue description.

Reviewer: I see you added a commit the other day, but this point at least hasn't changed. The list should have  six bullet points, but right now it has 10

> * If a bug, instructions to reproduce are included in the issue description.

Author: 6 bullet points of what?
It has 11 now, and I don't understand where to break the first or last 6 from the rest. Looks fine to me.

- The *QA-Team* labels it **needs confirmation** and tries to work with the user to confirm the bug and add potentially missing info to the report. Once the report is complete a **team-label** is added to the issue, the issue will be put on the **awaiting-triage-list** of the project and the label needs confirmation will be replaced with **bug**.
- The affected *Team* triages the issue and either rejects (label **wontfix**, closes and removes from the project) or accepts the issue. The team decides whether to fix the issue in-house (move to "Accepted" and assign a team member) or wait for a community developer to pick it up (by labeling it **help wanted**, removing it from the project board and providing hints on how to resolve the issue and further details on how the fix should be implemented if necessary). For non-critical issues trivial to fix, the label **good first issue** can be added as well.
- Once a *developer* is assigned and starts working on the issue, it is moved to "In Development". As soon as there is something to show for, the issue is closed and automatically moved to "Quality Assurance" for feedback from the QA team.

> If necessary, the developer provides hints on how to test the patch in a comment on the issue.

Reviewer: You insert a new line here. In general I am happy to see paragraphs of text broken into separate lines in the rst code. It makes it easier to see small changes in a diff if it's one sentence per line rather than a paragraph per line. 

Two points to this:
* here you do it inside an itemized list. Once you fixed the Jenkins build, please ensure that the itemized list still renders correctly and doesn't get broken by the newlines. I think it should be ok, but better safe than sorry
* provided that it IS ok to insert linebreaks, maybe you also want to do it in the other items in this list. you're touching every line already anyway, so review/diff already includes everything ... the best moment to take the hit and introduce the linebreaks

Author: Linebreaks are in. Build breaks not sure.

> - The *QA-Team* tests the fix on all devices and provides feedback to the developer. If problems are found, the issue is re-opened and goes back to "Accepted", elsewise it’s moved to "Release Candidate" to be included in the next release.

Author: moved to "rc"?

Reviewer: spelled out is fine

Author: Yes, but I think I may have gotten it wrong.
"it is elsewise moved to "Release Candidate" for inclusion come next release."

> - The *QA-Team* tests the fix on all devices and provides feedback to the developer. If problems are found, the issue is re-opened and goes back to "Accepted", elsewise it’s moved to "Release Candidate" to be included in the next release.

Reviewer: Is elsewise a word? how about otherwise?

- The *QA-Team* labels it **needs confirmation** and tries to work with the user to confirm the bug and add potentially missing info to the report.
- Once the report is complete a **team-label** is added to the issue, the issue will be put on the **awaiting-triage-list** of the project and the label needs confirmation will be replaced with **bug**.
- The affected *Team* triages the issue and either rejects (label **wontfix**, closes and removes from the project) or accepts the issue.
- The team decides whether to fix the issue in-house (move to "Accepted" and assign a team member) or wait for a community developer to pick it up (by labeling it **help wanted**, removing it from the project board and providing hints on how to resolve the issue and further details on how the fix should be implemented if necessary). For non-critical issues trivial to fix, the label **good first issue** can be added as well.
- Once a *developer* is assigned and starts working on the issue, it is moved to "In Development".
- As soon as there is something to show for, the issue is closed and automatically moved to "Quality Assurance" for feedback from the QA team.

> - If necessary, the developer provides hints on how to test the patch in a comment on the issue.

Author: Helps readability, but shouldn't be broken up into new points.

- The *QA-Team* tests the fix on all devices and provides feedback to the developer.

> - If problems are found, the issue is re-opened and goes back to "Accepted", otherwise it is moved to "Release Candidate" for inclusion in the next release.

Reviewer: same, please keep this sentence part of the previous bullet point
